cinder-{{cookiecutter.driver_name_lc}}/osci.yaml

```diff
@@ -0,0 +1,8 @@
+- project:
+    templates:
+      - charm-unit-jobs-py38
+      - charm-unit-jobs-py39
```

Comment:
This doesn't run Zaza at all, see https://review.opendev.org/c/openstack/charm-cinder-nimblestorage/+/825162

Reply:
discussed with @ChrisMacNaughton: this is probably OK as usually when creating a new cinder charm, we won't have a backend appliance ready in our CI system at first

Reply:
ideally, part of accepting a new charm that has been written with this cookiecutter is documented manual validation that _has_ been run against the hardware in question, though :)